packages/protocol/contracts/governance/Validators.sol

```diff
@@ -1076,4 +1076,14 @@ contract Validators is
     emit ValidatorDeaffiliated(validatorAccount, affiliation);
     return true;
   }
+
+  /**
+   * @notice Removes a validator from the group for which it is a member.
+   * @param validatorAccount The validator to remove and deaffiliate from their affiliated validator group.
+   */
+  function removeSlashedMember(address validatorAccount) external {
+    Validator storage validator = validators[validatorAccount];
```

Comment:
What happens if the account is not a validator?

Reply:
Following the precedent of `deaffiliate` I `require` here, but I'm still unsure on the exact assumptions on callers of this fn, so maybe this should be a simple if

Reply:
Yeah I would actually consider making this a no-op if the account is not a validator, and calling it `forceDeaffiliationIfValidator`.

If we require that the account is a validator, the Downtime and Double signing slashers will need to check that the account they're slashing is a validator before calling into this function, which is not as efficient as calling into `Validators` once and letting it decide whether or not the deaffiliation needs to take place